release.yaml

```diff
@@ -21,6 +21,9 @@ internal:
   create:
     steps:
       patch:
+          - name: "gh:auth"
+            cmd: |
+              gh auth status
```

Comment:
I think we should instead move this to the `requirements` and instead call `checkDeps` in `(*releaseRunner) Test(context.Context) error`. The repetition here is very error prone.

Reply:
https://github.com/sourcegraph/sourcegraph/pull/62143